homework-1/src/repository/DayOfMonth.java

```diff
@@ -0,0 +1,50 @@
+package repository;
+
+import java.util.Scanner;
+
+public class DayOfMonth {
+	/*
+	 * Se introduce luna si anul de la tastatura si 
+	 * in functie de aceste inputuri se alege mesajul
+	 * de afisat
+	 */
+	public void zilenLuna() {
+		Scanner scanner = new Scanner(System.in);

```

Comment:
not ok
separate input from calculation logic, calculation logic should be reusable regardless input context